src/db/mongo/MongoInterface.cpp

```diff
@@ -81,10 +81,15 @@ MongoInterface::MongoInterface()
 	bson_error_t err;
 	std::string mongo_uri;
 	if(!rosprolog_kb::node().getParam(std::string("mongodb_uri"),mongo_uri)) {
-		char *mongo_host = std::getenv("MONGO_PORT_27017_TCP_ADDR");
+		char *mongo_host = std::getenv("NEEMHUB_MONGO_HOST");
 		if(mongo_host != NULL) {
+			char *mongo_user_name = std::getenv("NEEMHUB_MONGO_USER");
```

Comment:
better: KNOWROB_MONGO_USER and so on

Reply:
I tried to keep the naming consistent with the names in the openease packages. Should we rather use KNOWROB_MONGO_USER everywhere or NEEMHUB_MONGO_USER everywhere?

Reply:
Other possibility would be REMOTE_MONGO_USER

Reply:
it might be most useful to have a "KNOWROB_" prefix.
First, then it is obvious to which project this belongs when looking at the env.
Second, there could be also other infrastructure then NEEM-Hub used as a backend for KnowRob.
Third, the env is global and we need to avoid name clashes so I would vote against "REMOTE_" prefix